src/cobalt.rs

```diff
@@ -120,6 +124,57 @@ pub fn build(config: &Config) -> Result<()> {
         }
     }
 
+    // load data files
+    let mut data_map : HashMap<String, Value> = HashMap::new();
+    let data_root = source.join("_data");
+    let data_files_builder = FilesBuilder::new(data_root.as_path())?;
+    let data_files = data_files_builder.build()?;
+
+    for df in data_files.files() {
+
+        if let Some(ext) = df.extension() {
+            if let Some(file_stem) = df.file_stem() {
+                let file_name = String::from(file_stem.to_str().unwrap());
+                let full_path = data_root.join(df.clone());
+                let data : Value;
+
+                if ext == OsStr::new("yml") || ext == OsStr::new("yaml")  {
+                    let reader = File::open(full_path)?;
+                    data = serde_yaml::from_reader(reader)?;
+                } else if ext == OsStr::new("json") {
+                    let reader = File::open(full_path)?;
+                    data = serde_json::from_reader(reader).unwrap();
+                } else if ext == OsStr::new("toml") {
+                    let mut reader = File::open(full_path)?;
+                    let mut text = String::new();
+                    reader.read_to_string(&mut text)?;
+                    data = toml::from_str(&text).unwrap();
+                } else {
+                    continue
```

Comment:
Probably should log or error on unsupported file types

Reply:
Will log.

Reply:
Is there a reason someone might want to have an unsupported file in here?

If so, we should leave it as a trace.  If not, we should make it a warning.